src/test/java/com/homework/MyArrayListTest.java

```diff
@@ -0,0 +1,51 @@
+package com.homework;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class MyArrayListTest {
+    ArrayList arrayList;
```

Comment:
add access modificator

Reply:
program to interfaces, not implementation
and better generelize your List with some type